### ISSUE TITLE
Fix `.mejs-controls` UI glitch when Legacy template is active

### ISSUE THREAD

**Issue by kjnanda** [rtMediaWP/rtMedia#1350]
When BuddyPress Legacy template has activated the alignment of player control buttons needs improvement. 

Please check this screenshot.

![Unnamed image](https://user-images.githubusercontent.com/15173772/58700457-b0379100-83bd-11e9-8197-62ff4722a701.png)

------

Solution - Can be fixed by setting the value for `bottom CSS` property with some value as shown in the screenshot`. Make sure that fix should equally work for any WordPress default theme.

![Gmail Screenshot by cloudHQ 5_31_2019 4_08_28 PM](https://user-images.githubusercontent.com/15173772/58700710-787d1900-83be-11e9-833e-d94afa256c4f.png)

-----

```
Test Environment 

1. Wordpress 5.2.1
2. Bussypress 4.3.0 - Legacy template
3. rtMedia 4.5.6
4. Theme - all wordpress default theme

```

**Comment by surajkrsingh** [comment]
PR:-https://github.com/rtMediaWP/rtMedia/pull/1383

**Comment by kjnanda** [comment]
Seems the issue has been fixed with other PR. UI looks good now.

**Comment by kjnanda** [comment]
This is working as expected and tested with `pre-release` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `dev` branch. ✅

**Comment by kjnanda** [comment]
This is working as expected and tested with `master` branch. ✅

**Comment by kjnanda** [comment]
Closing this issue as this has been fixed and release in - V 4.5.8
